signac/contrib/migration/v1_to_v2.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2022 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+"""Migrate from schema version 1 to version 2.
+
+This migration involves the following changes:
```

Comment:
Probably need to fill out some bullet points here.

Reply:
I originally had something here but figured I would update it at the end. No harm in building it up incrementally though.